test/functional/android/appium_tests.py

```diff
@@ -196,9 +206,19 @@ def test_open_notifications(self):
     def test_set_text(self):
         self.driver.find_element_by_android_uiautomator(
             'new UiScrollable(new UiSelector().scrollable(true).instance(0)).scrollIntoView(new UiSelector().text("Views").instance(0));').click()
-        self.driver.find_element_by_name('Controls').click()
-        self.driver.find_element_by_name('1. Light Theme').click()
-
+        WebDriverWait(self.driver, 3).until(
```

Comment:
Might be able to remove wait here since below comments already exists.

https://github.com/appium/python-client/pull/364/files#diff-349bc3f80ce233a0a57b4489372b3894R35

Reply:
Use SLEEPY_TIME instead of ```3```